SEO: reword introduction paragraph into descriptions - Part 1 (#408)
front-commerce/developers.front-commerce.com: source/docs/advanced/theme/wysiwyg-platform.md

---
id: wysiwyg-platform
title: WYSIWYG customization

> description: In this guide you will learn the different kind of WYSIWYG that is available on Front-Commerce and how to customize them.

Reviewer: ```suggestion
description: In this guide, you will learn the different WYSIWYG implementations available on Front-Commerce and how to customize them.
```
